packages/precommit-hooks/default-hooks.yaml

```diff
@@ -0,0 +1,48 @@
+# Pre-commit hook configuration file
+#
+# This contains the list of hooks to be run before committing code.
+# The devsecops-hooks baseline hook is temporarily disabled while we investigate issues with it but will eventually become compulsory.
+# Other hooks maybe added or removed as needed to suit individual project requirements.
+
+repos:
+  #. Temporary disabled while we investigate issues with it
+  #  - repo: https://github.com/ministryofjustice/devsecops-hooks
+  #    rev: v1.1.0
+  #    hooks:
+  #      - id: baseline
+  - repo: local
+    hooks:
+      - id: gitleaks
+        name: Scan commit for secrets
+        language: system
+        entry: gitleaks git --pre-commit --redact --staged --verbose --config .gitleaks/config.toml --gitleaks-ignore-path .gitleaks/.gitleaksignore
+        require_serial: true
+        pass_filenames: false
+      - id: lint
+        name: linting code
+        language: system
+        entry: npm run lint
+        types: [ts]
```

Comment:
this is nicer than husky as it skips the check if no typescript files are included